integrated_channels/api/v1/sap_success_factors/urls.py

```diff
@@ -0,0 +1,12 @@
+"""
+    url mappings for integrated_channels/v1/sap_success_factors/
+"""
+
+from rest_framework import routers
+
+from .views import SAPSuccessFactorsConfigurationViewSet
+
+app_name = 'sap_success_factors'
+router = routers.DefaultRouter()  # pylint: disable=invalid-name
```

Comment:
I'm sure it's necessary, but why are we having to disable the invalid-name rule?

Reply:
Upon running tox with this comment removed it doesn't actually appear to be necessary.
This was only just copypasta from other places.
It's likely either this rule applied to edx-platform but somehow not edx-enterprise OR whatever rule we used when this was first implemented from wherever I copied it is either explicitly ignored or no longer applies. I'll just try committing the removal.